flocker/docs/bootstrap/_extension.py

```diff
@@ -0,0 +1,60 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Bootstrap extension entry point.
+"""
+
+from docutils import nodes
+from sphinx.writers.html import HTMLTranslator
+from . import _simple, _tabs
+
+
+def setup(app):
+    """
+    Entry point for sphinx extension.
+    """
+
+    for module in [_simple, _tabs]:
+        module.setup(app)
+
+
+class HTMLWriter(HTMLTranslator):
+    """
+    Overrides part of the default HTMLTranslator to provide specific
+    class names on some generated HTML elements.
```

Comment:
I think you probably need to add docutils and sphinx copyright lines to the top of this file.